src/hooks/useOnClickOutside.ts

```diff
@@ -1,18 +1,24 @@
 import { useEffect } from "react";
 
 export const useOnClickOutside = (
-  ref: React.RefObject<HTMLElement>,
-  cb: () => void
+  refs: Array<React.RefObject<HTMLElement>>,
+  cb: (isFocused: boolean) => void,
+  isOpen: boolean
```

Comment:
I think as a generic hook it shouldn't case on specific use cases. This implies it can only be used for some openable element.  

A suggestion would be the implementer should make their callback case on this condition.